CRM/Lineitemedit/Form/Edit.php

```diff
@@ -175,7 +175,7 @@ public function submit($values, $isTest = FALSE) {
     );
 
     $lineItem = CRM_Price_BAO_LineItem::create($params);
-    $lineItem = $lineItem->toArray();
+    $lineItem = civicrm_api3('LineItem', 'getsingle', ['id' => $this->_id]);
```

Comment:
The test `CRM_Lineitemedit_Form_EditTest::testParticipantRecordOnLineItemEdit` was failing because both entity_id and entity_table weren't included in the update so its easier to call get single to get all the data so that we pass through to updateEntityRecord function